hq/app/utils/attempt/Attempt.scala

```diff
@@ -155,4 +160,47 @@ object Attempt {
     def Left[A](ferr: Future[FailedAttempt])(implicit ec: ExecutionContext): Attempt[A] =
       Attempt(ferr.map(scala.Left(_)))
   }
+
+
+
+
+  /**
+    * @see https://stackoverflow.com/questions/16359849/scala-scheduledfuture
+    * */
+  object Delayed {
```

Comment:
The formatting is off on this block because it's been pasted from SO. Can you make it consistent with the codebase?